api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -153,7 +153,7 @@ public void checkFreeTierBillingUsage_exceedsDollarThresholds() throws Messaging
     // check that we do not alert twice for the 50% threshold
 
     freeTierBillingService.checkFreeTierBillingUsageForUsers(Sets.newHashSet(user), allBQCosts);
-    verifyZeroInteractions(mailService);
+    verifyNoMoreInteractions(mailService);
```

Comment:
`verifyNoInteractions` fails here because `mailService` is used above. "NoMore" passes and seems accurate.